helpers/build_packages.sh

```diff
@@ -294,7 +295,7 @@ if [ "$BUILDGG" = "1" ]; then
     # look for either DEVICE or HABUILD_DEVICE files, do not use wildcards as there could be other variants
     pattern_lookup=$(ls "$ANDROID_ROOT"/hybris/droid-configs/patterns/jolla-hw-adaptation-{$DEVICE,$HABUILD_DEVICE}.yaml 2>/dev/null | uniq)
 
-    if grep -q "^- gstreamer1.0-droid" "$pattern_lookup" &>/dev/null; then
+    if grep -q "^- gstreamer1.0-droid" "$pattern_lookup" &>/dev/null || grep -q "^- gmp-droid" "$pattern_lookup" &>/dev/null; then
```

Comment:
You can use grep -q -s to avoid redirecting stderr to /dev/nulll.

Reply:
Replaced all these in second commit.

Reply:
I've pushed your cca6d97 cleanup commit straight to master, because it's just that good ;) (and other PR is already using it).
You can remove the second commit of yours from this PR.